manifests/c/composer/composer/2.3.5/composer.composer.installer.yaml

```
@@ -0,0 +1,44 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.5-1-19041-1645
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: composer.composer
+PackageVersion: 2.3.5
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+InstallModes:
+- interactive
+UpgradeBehavior: install
+Commands:
+- composer
+ReleaseDate: 2022-04-13
+Installers:
+- Architecture: x64
+  InstallerUrl: https://getcomposer.org/Composer-Setup.exe
+  InstallerSha256: 10826B503BA6823F14893F0627823017EE1778881222EA9960A7E268739D6956
+  InstallerSwitches: 
+    Custom: /CURRENTUSER
+    Silent: /VERYSILENT /SUPPRESSMSGBOXES
```

```suggestion
    Silent: /VERYSILENT /SUPPRESSMSGBOXES
```
This isn't really needed here because WinGet will use the correct switches since the `InstallerType` is set to `inno`.

WinGet uses `/VERYSILENT` for `Silent:` and `/SILENT` for `SilentWithProgress`